index.js

```diff
@@ -5,6 +5,7 @@ var p       = require('path');
 
 var _       = require('lodash');
 var yaml    = require('js-yaml');
+var merge   = require('merge');
```

Comment:
No microdeps please, `lodash` already has `assign()` and other similar functions.